src/ex1/style.css

```diff
@@ -0,0 +1,163 @@
+* {
+    font-family: Alef, Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 100%;
+    background: linear-gradient(to top, rgb(47, 255, 102) 0%, rgb(0, 200, 255) 100%);
+}
+
+h1 {
+    font-size: 35px;
+}
+
+
+.container {
+    background-color: white;
+    border-radius: 10px;
+    width: 100%;
+    max-width: 500px;
+    padding: 25px;
+}
+
+.btn[disabled] {
+    opacity: 0.3;
+    box-shadow: none;
+    border: none;
+}
+
+.btn[disabled]:hover {
+    cursor: not-allowed;
+    box-shadow: none;
+}
+
+.btn {
+    height: 100%;
+    text-align: center;
+    color: white;
+    border: none;
+    margin-left: 10px;
+    border-radius: 8px;
+    cursor: pointer;
+    background-color: rgb(30, 75, 254);
+}
+
+.btn:hover {
+    background-color: rgb(0, 51, 255);
+    box-shadow: 0.5px 0.5px 10px 0.5px grey;
+}
+
+
+.placeHolder {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    color: greenyellow;
+    font-size: 100px;
+    opacity: 0.3;
+    padding: 60px;
+    border: 1px solid rgb(99, 99, 99);
+    border-radius: 8px;
+}
+
+.hidden {
+    display: none;
+}
+
+.placeHolder p {
+    margin-top: 5px;
+    color: black;
+}
+
+#addBtn {
+    width: 60px;
+    font-size: 30px;
+}
+
+#add-task {
+    display: flex;
+    width: 100%;
+    height: 50px;
+    margin: 30px 0;
+}
+
+#add-task input {
+    width: 90%;
+    height: 100%;
+    border: 1px solid #ccc;
+    border-radius: 8px;
+    padding-left: 15px;
+    outline: none;
+}
+
+
+#taskList {
+    max-height: 260px;
+    overflow-y: auto;
+}
+
+
+#taskList li {
```

Comment:
Using "li" here instead of a class makes it coupled to this specific solution. If you'll need to change li to a different element (ex: div) you'll have to make many changes in the code